esp/services/ws_dfu/ws_dfuService.cpp

```diff
@@ -5957,31 +5954,36 @@ void CWsDfuEx::getFileMeta(IPropertyTree &metaInfo, IDistributedFile &file, IUse
 #ifdef _USE_OPENSSL
     if (!isEmptyString(keyPairName)) // without it, meta data is not encrypted
     {
-        metaInfo.setProp("keyPairName", keyPairName);
-
-        Owned<IPropertyTree> secureMetaInfo = createPTreeFromIPT(&metaInfo);
-        secureMetaInfo->setProp("expiryTime", expiryTime);
-
-        extractFilePartInfo(*secureMetaInfo, *fDesc);
-
-        // 1st serialize to JSON
-        StringBuffer secureMetaInfoJson;
-        toJSON(secureMetaInfo, secureMetaInfoJson);
-
-        // 2nd compress
-        MemoryBuffer compressedSecureMetaInfoMb;
-        fastLZCompressToBuffer(compressedSecureMetaInfoMb, secureMetaInfoJson.length(), secureMetaInfoJson.str());
-
-        // 3rd encrypt
-        const char *publicKeyFName = env->getPublicKeyPath(keyPairName);
-        Owned<CLoadedKey> publicKey = loadPublicKeyFromFile(publicKeyFName, nullptr);
-        MemoryBuffer encryptedSecureMetaInfoMb;
-        aesEncryptWithRSAEncryptedKey(encryptedSecureMetaInfoMb, compressedSecureMetaInfoMb.length(), compressedSecureMetaInfoMb.bytes(), *publicKey);
-        metaInfo.setPropBin("secureInfo", encryptedSecureMetaInfoMb.length(), encryptedSecureMetaInfoMb.bytes());
+        metaInfo->setProp("version", securityInfoVersion);
+        metaInfo->setProp("logicalFilename", file.queryLogicalName());
+        metaInfo->setProp("jobId", req.getJobId());
+        metaInfo->setProp("accessType", req.getAccessTypeAsString());
+        StringBuffer userStr;
+        if (user)
+            metaInfo->setProp("user", user->getUserName(userStr).str());
+        metaInfo->setProp("keyPairName", keyPairName);
+        metaInfo->setProp("expiryTime", expiryTime);
```

Comment:
Could spark (or any other client) potentially want to know any of this information?

Currently it is serialized, compressed and converted to base64.  It makes it very hard to read, is it significantly smaller than the json for the metaInfo?

Reply:
It is also in the Esp response.